app/src/main/java/com/github/studydistractor/sdp/event/EventViewModel.kt

```diff
@@ -0,0 +1,121 @@
+package com.github.studydistractor.sdp.event
+
+import androidx.lifecycle.ViewModel
+import com.github.studydistractor.sdp.ui.state.EventUiState
+import com.google.android.gms.tasks.Task
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+
+class EventViewModel(eventModel: EventModel): ViewModel() {
+    private val _eventModel: EventModel = eventModel
```

Comment:
Why don't we juste use `eventModel`, is there a reason for this ?

Reply:
Actually no! I'll remove them in this pr